sectionsUtilityTool/src/main/webapp/assets/js/filters/filters.js

```diff
@@ -1,12 +1,34 @@
 'use strict';
 /* jshint  strict: true*/
-/* global angular */
+/* global angular, _ */
 
-
-angular.module('sectionsFilters', []).filter('filterName', function () {
-  return function () {
+//mot used - but keep for reference
+angular.module('sectionsFilters', []).filter('filterMedSchool', function () {
+  return function (items) {
+  	var filtered = [];
+    for (var i = 0; i < items.length; i++) {
+      var item = items[i];
+      if (item.sis_term_id !==null) {
+		      if (item.sis_term_id.slice(-1) === '0') {
+		        filtered.push(item);
+	  	    }
+      }
+    }
+    return filtered;
   };
-}).filter('anotherFilter', function () {
-  return function () {
+}).filter('showOnlyTermsWithCourses', function () {
+  return function (terms) {
+    var termsArray = [39,80,1];
```

Comment:
Hai - this filter is not used. Neither is 'anotherFilter'

Will need some severe clean up soon.